OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -123,6 +123,17 @@ static void BindSliderPref(Widget parent, string id, object group, string pref)
 			ss.OnChange += x => field.SetValue(group, x);
 		}
 
+		static void BindExpSliderPref(Widget parent, string id, object group, string pref)
```

Comment:
This looks identical to `BindSliderPref` which I expect should already do what you need because `ExponentialSliderWidget` is a subclass.  This method should be able to go away.